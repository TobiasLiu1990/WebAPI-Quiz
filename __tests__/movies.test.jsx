
//describe är ett set/suite med tester
describe("quiz pages", () => {

    //it - ett test för settet.
    it("quiz list", () => {
        expect("quiz").toBe("quiz");           //expert == assert i java.
    });


    it("lets user add a new quiz", () => {

    });

});