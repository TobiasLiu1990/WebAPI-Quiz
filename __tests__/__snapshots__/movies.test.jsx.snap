// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quiz pages quiz list 1`] = `"<div><h1> The Quiz </h1><div><h2>The country of Pizza</h2><button value="1">Norway</button><button value="2">Italy</button><button value="3">Spain</button><button value="4">Germany</button><hr></div><div><h2>The country of Sushi</h2><button value="1">Finland</button><button value="2">New Zealand</button><button value="3">Japan</button><button value="4">China</button><hr></div><div><h2>The country of Taco</h2><button value="1">Spain</button><button value="2">Portugal</button><button value="3">Chile</button><button value="4">Mexico</button><hr></div><div><h2>The country of Smalahove</h2><button value="1">Norway</button><button value="2">Sweden</button><button value="3">Denmark</button><button value="4">Finland</button><hr></div><br><h2 id="print-answer"></h2></div>"`;
